pkgs/applications/misc/nb/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, lib, fetchFromGitHub, installShellFiles
+, bash, git
+}:
+stdenv.mkDerivation rec {
+  version = "5.7.8";
+  pname = "nb";
+
+  src = fetchFromGitHub {
+    owner = "xwmx";
+    repo = "nb";
+    rev = version;
+    sha256 = "1sw09sgskzsn3dy4ipvl52hwh5k3cwqfhjnm7rmxbzcpfk3jm3sp";
+  };
+
+  buildInputs = [ bash git ];
+  nativeBuildInputs = [ installShellFiles ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin/
+    mv nb $out/bin/
+    mv bin/bookmark $out/bin/
+  '';
+
+  postInstall = ''
+    installShellCompletion etc/nb-completion.{bash,zsh}
```

Comment:
That is not run because you have overwritten installPhase without running pre and postInstall.

Reply:
Thank you, missed this completely!
Do nix-shells inherit completions? Don't seem to be able to test them.